src/middleware/errorHandler.ts

```
@@ -4,14 +4,19 @@ import { logger } from "../config/logger";
 export class AppError extends Error {
   statusCode: number;
   code: string;
-  details?: any;
   isOperational: boolean;
   details?: unknown;
 
-  constructor(message: string, statusCode: number, details?: unknown) {
+  constructor(message: string, statusCode: number, codeOrDetails?: string | unknown, details?: unknown) {
```

ESLint

[error] Replace `message:·string,·statusCode:·number,·codeOrDetails?:·string·|·unknown,·details?:·unknown` with `⏎····message:·string,⏎····statusCode:·number,⏎····codeOrDetails?:·string·|·unknown,⏎····details?:·unknown,⏎··`

(prettier/prettier)